infrastructure/deploy/main.tf

```diff
@@ -229,6 +237,10 @@ resource "aws_s3_bucket_lifecycle_configuration" "meadow_streaming" {
     filter {
       prefix = ""
     }
+    
+    noncurrent_version_expiration {
+      noncurrent_days = 30
```

Comment:
@bmquinn Let's set this at 90 days. I don't know why I initially called out 30 days. 90 gives us a nicer cushion.